tag/nitag.yml

```diff
@@ -0,0 +1,1749 @@
+    get:
+      tags:
+        - subscriptions
+      summary: Get the current values for the tags in a subscription
+      description: Returns the current values for the tags in the subscription. Can return either all values or only new values since the last request.
+      operationId: GetSubscriptionTagValues
+      parameters:
+        - in: path
+          name: id
+          description: Subscription ID
+          type: string
+          required: true
+      responses:
+        200:
+          $ref: '#/responses/GetSubscriptionUpdatesResponse'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/tags-rpc/subscriptions/heartbeat:
+    post:
+        tags:
+          - subscriptions
+        summary: Update the heartbeat timer of multiple subscriptions
+        description: Updates the heartbeat timers of multiple subscriptions. This route is deprecated in favor of /v2/refresh-subscriptions.
+        operationId: RpcUpdateSubscriptionHeartbeats
+        deprecated: true
+        parameters:
+          - in: body
+            name: subscriptionIds
+            description: IDs of the subscriptions whose heartbeat timers you want to update
+            required: true
+            schema:
+              type: object
+              properties:
+                subscriptionIds:
+                  type: array
+                  items:
+                    type: string
+        responses:
+          200:
+            description: Success
+            schema:
+              description: Update subscriptions heartbeat response
+              type: object
+              title: Update Subscriptions Heartbeat Response
+              properties:
+                invalidSubscriptionIds:
+                  description: List of invalid subscription IDs
+                  type: array
+                  items:
+                    type: string
+          401:
+            $ref: '#/responses/Unauthorized'
+          default:
+            $ref: '#/responses/Error'
+  /v2/refresh-subscriptions:
+    post:
+      tags:
+        - subscriptions
+      summary: Update the heartbeat timer for multiple subscriptions (SystemLink Server only)
+      description: Updates the heartbeat timers for multiple subscriptions
+      operationId: UpdateSubscriptionHeartbeats
+      parameters:
+        - in: body
+          name: subscriptionIds
+          description: IDs of the subscriptions whose heartbeat timers you want to update
+          required: true
+          schema:
+            type: object
+            properties:
+              subscriptionIds:
+                type: array
+                items:
+                  type: string
+      responses:
+        200:
+          description: Success
+          schema:
+            type: object
+            title: Update Subscriptions Heartbeat Response
+            properties:
+              invalidSubscriptionIds:
```

Comment:
Ideally, we should make PartialSuccessResposne for the partial successes. However, this document has to resemble the existing implementations of the APIs, no change is required, but making this response to PartialSuccessResponse should be tracked by some CAR or user story.

Reply:
I filed 712984